tracetool/README.md

```diff
@@ -0,0 +1,70 @@
+# Linux trace tool
+
+
+## Introduction
+
+This is python script tool to analysis Linux ftrace, support trace events from
+Linux kernel(e.g. i915) and from user mode lib/application(e.g. libva, media driver).
+
+
+## Linux trace capture
+
+1. Install trace-cmd::
+
+    `sudo apt-get install trace-cmd`
+
+2. Grant write access to trace node for application::
+
+    `sudo chmod 777 /sys/kernel/debug/`  
+    `sudo chmod 777 /sys/kernel/debug/tracing/`  
+    `sudo chmod 777 /sys/kernel/debug/tracing/trace_marker_raw`
+
+3. Enable libva trace::
+
+    `export LIBVA_TRACE=FTRACE`  
+    to capture buffer data  
+    `export LIBVA_TRACE_BUFDATA=1`
+
+4. Run application under trace-cmd::
+
+    In a proxy mode:  
+    `trace-cmd record -e i915 <workflow-cmd-line>`
+
+5. Output is "trace.dat"
+
+Alternatively you can collect trace data in separate terminal.  
+It is useful if you want to profile daemon or a service:
+
+1. Start trace capture::
+
+    `sudo trace-cmd record -e i915`
+
+2. Run test app in another terminal
+3. Stop capturing in the first terminal
+4. Output is "trace.dat"
+
+
+## Trace post-processing and analysis
+
+    python3 main.py [-raw] file.dat|file.etl [file.dat|file.etl ...]
+
+Options:
+
+* ``-raw`` - Parse trace events and dump into <trace-file>.csv file (ftrace) or <trace-file>.json file (GUC)
```

Comment:
What this GUC comment is about? If you don't support parsing GUC logs as of now - remove that from readme.

Reply:
removed. open source tool don't support guc log process